backend/agents/agent.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC
+from typing import List, Type
+from utils import call_model_with_tools
+from .adapters import get_mistral_tool

```

Comment:
[Import comment from before](https://github.com/WaitThatShouldntWork/InferGPT/pull/39/files#r1584566963)

Reply:
yeah I like it, will change. Although as I am doing this, is there a subtle difference between what should be in the init file and locally imported i.e. is the init file exposing to other modules what is available from this one? Whereas there maybe some functionality that we don't want other modules to use? (me thinking out loud, happy to change to init file approach for now with all as like it)

Reply:
This is true and something I wasn't thinking about, if outside the /agents directory it shouldn't be accessible then lets keep it out of the __init__ file and import relatively (as that's appropriate as all the files that want that method are _relatively_ close to it)